conductr_cli/conduct_main.py

```diff
@@ -365,7 +364,8 @@ def run(_args=[]):
             else:
                 args.local_connection = False
 
-        args.cli_parameters = get_cli_parameters(args)
+            args.cli_parameters = get_cli_parameters(args)
+
         args.custom_settings = get_custom_settings(args)
```

Comment:
The custom settings are not necessary for the `conduct version` command. So let's move it into the above if clause